sdk/storage/azdatalake/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.2 (Unreleased)
+## 1.1.2-beta.1 (2024-04-10)
```

Comment:
Need to update https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/storage/azdatalake/internal/exported/version.go#L11 to match.

Reply:
Updated, thanks!